### PR TITLE
VZ-6723: Document the configuration of a database for Grafana

### PR DESCRIPTION
**Pull request by mgianatagh**
Add doc for the configuration of a database for Grafana

### CODE REVIEW

File: content/en/docs/setup/customizing/grafana.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  For example, you could configure it as shown:
+
+  ```
+  $ # Load the login credentials into variables
+  $ ROOT_SECRET=$(echo <database root user secret> | base64)
+  $ USER=$(echo <database user> | base64)
+  $ USER_SECRET=$(echo <database user secret> | base64)
+  $ #
+  $ # Create the secret
+  $ kubectl apply -f - <<-EOF
```
Comment (pfmackin) [inline content/en/docs/setup/customizing/grafana.md:27]: I cut and pasted this and it didn't work.

File: content/en/docs/setup/customizing/grafana.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+Weight: 9
+draft: false
+---
+
+
+### External Grafana Database
+
+By default, Verrazzano automatically installs and configures a Grafana database, you have the option to use your own external database.  
+
+If you want to provide your own Grafana database , you must:
```
Comment (pfmackin) [inline content/en/docs/setup/customizing/grafana.md:14]: Are the 2 options below mutually exclusive?  If so mention that please

File: content/en/docs/setup/customizing/grafana.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+* Create a secret named `grafana-db` in the `verrazzano-install` namespace which contains the login credentials.
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply content/en/docs/setup/customizing/grafana.md:16]: yes

File: content/en/docs/setup/customizing/grafana.md (hunk 2 above)
Comment (mgianatagh) [reply content/en/docs/setup/customizing/grafana.md:14]: no, both are required.  I changed the text to say "you must do the following steps"

File: content/en/docs/setup/customizing/grafana.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    username: $USER
+    password: $USER_SECRET
+  EOF
+  ```
+* Configure the Grafana component of the Verrazzano custom resource.
+
+  For example, you could configure it as shown:
+
+  ```
+  apiVersion: install.verrazzano.io/v1alpha1
```
Comment (anders-swanson) [inline content/en/docs/setup/customizing/grafana.md:45]: Please use v1beta1 in master branch

Comment (mgianatagh) [reply content/en/docs/setup/customizing/grafana.md]: done

File: content/en/docs/reference/API/Verrazzano/Verrazzano.md [hunk 5]
```diff
@@ -326,9 +326,18 @@ To configure OpenSearch, instead of using install args, Oracle recommends that y
 | `enabled` | Boolean | If true, then Prometheus will be installed. Defaults to `true`. This is a legacy setting; the preferred way to configure Prometheus is using the [prometheusOperator](#prometheus-operator-component) component. | No |
 
 ### Grafana Component
```
[the remark this answers is not in the document]
Comment (mgianatagh) [reply content/en/docs/reference/API/Verrazzano/Verrazzano.md]: done

**Review by pfmackin — APPROVED** [review]

**Review by anders-swanson — APPROVED** [review]

File: content/en/docs/setup/customizing/grafana.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+  spec:
+    profile: dev
+    components:
+      grafana:
+        database:
+          host: mysql.verrazzano-install.svc.cluster.local
+          name: grafana
+  ```
+
+Refer to the table in the Verrazzano custom resource pertaining to the [Grafana component]({{< relref "/docs/reference/API/Verrazzano/v1beta1.md#grafana-component" >}}) for further details of the component definition.
```
Comment (MelwynPaul) [inline content/en/docs/setup/customizing/grafana.md:58]: Please update - 

From: Refer to the table in the Verrazzano custom resource pertaining to the [Grafana component]({{< relref "/docs/reference/API/Verrazzano/v1beta1.md#grafana-component" >}}) for further details of the component definition./

To: For more information about the component definition, see [Grafana component]({{< relref "/docs/reference/API/Verrazzano/v1beta1.md#grafana-component" >}}) in the Verrazzano custom resource.

**Review by MelwynPaul — COMMENTED** [review]
Added a few comments.

**Review by MelwynPaul — APPROVED** [review]
Verified. Looks good. 
Approved.

**Pull request merged**